reference-implementation/lib/transform-stream.js

```diff
@@ -208,7 +233,8 @@ class TransformStreamSink {
 
     transformStream._writableDone = true;
 
-    const flushPromise = PromiseInvokeOrNoop(transformStream._transformer, 'flush', [transformStream._controller]);
+    const flushPromise = PromiseInvokeOrNoop(transformStream._transformer,
+                         'flush', [transformStream._transformStreamController]);
```

Comment:
This is inconsistent with transformPromise above, where a temporary variable called "controller" is created first. Also startResult, below.